docs/spec_draft.md

```diff
@@ -1267,6 +1268,145 @@ More formally, `result[i0, ..., iR-1]` is equal to:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.reduce_window
+
+### Semantics
+
+Performs a reduction function `body` over all elements in each window of a
+sequence of N `inputs` tensors and produces a sequence of N `results` tensor.
+
+The following diagram shows how elements in `results[k]` are computed from
+`operand[k]` using a concrete example.
+
+<p align="center">
+  <img src="spec_draft/reduce_window.svg" />
```

Comment:
One last change, similar to how we specify `input[k]`, can you do the same for `input_value = 0` -> `input_values[k] = 0` on the right and also in the 4 calculations at the bottom?

Reply:
Thanks for the catch! done